### PR TITLE
Feature section

### PR DESCRIPTION
**Pull request by ahmedsaleh10**

### CODE REVIEW

File: src/components/ProductDetails/ProductDetails.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+      const product = products.find(item => item.id === parseInt(productId) )
+      console.log(product)
+      return(
+       <>{product.description}</>
+    )
+    
+    }
+    else{
+        return(<Loading/>)
+    }
```
Comment (soraabunasser1993) [inline src/components/ProductDetails/ProductDetails.js:18]: as we discussed please replace this into ternary check

File: src/components/ProductCard/ProductCard.js [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+          <Typography variant="body1" className={classes.DeletedPrice}>
+            ${(100 * product.price) / (100 - 100 * product.salePercantage)}
+          </Typography>
+          <Typography variant="body1"> ${product.price} </Typography>
+        </Stack>
+      ) : ""
+      }
+
+      {!product.sale ? (
+        <Stack
```
Comment (soraabunasser1993) [inline src/components/ProductCard/ProductCard.js:46]: this should be implemented at the line 42

File: src/components/Features/Features.js [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+        <Grid item className={classes.GridItem} xs={6} lg={3}>
+            <Feature icon={<WatchOutlinedIcon color='error'/>} title={"24/7 Help Center"} description={"support online 24hours"}/> 
+        </Grid>
+        <Grid item className={classes.GridItem} xs={6} lg={3}>
+         <Feature icon={<CardGiftcardOutlinedIcon color='error'/>} title={"Gift Promotion"} description={"an orders over $49.00"}/>
+        </Grid>
+        <Grid item className={classes.GridItem} xs={6} lg={3}>
+         <Feature icon={<AttachMoneyOutlinedIcon color='error'/>} title={"Money Gurantee"} description={"15 days money back"}/>
+        </Grid>
+    </Grid>
```
Comment (soraabunasser1993) [inline src/components/Features/Features.js:26]: maybe making the data as an array will be more clear and fewer lines of code

File: src/components/ProductCard/ProductCard.js (hunk 2 above)
Comment (ahmedsaleh10) [reply src/components/ProductCard/ProductCard.js:46]: I change it to ternary check in Best Sales branch

**Pull request merged**